packages/bytebot-agent/src/main.ts

```diff
@@ -13,11 +13,17 @@ async function bootstrap() {
   try {
     const app = await NestFactory.create(AppModule);
 
+    // Set global prefix for all routes
+    app.setGlobalPrefix('api');
```

Comment:
I'm not sure how the the current /api routes are working I sorta needed this but feels wrong